src/index.js

```diff
@@ -1,13 +1,19 @@
+/* eslint-disable no-unused-vars */
 /* eslint comma-dangle: ["error", "never"] */
+
 import React from 'react';
 // eslint-disable-next-line
```

Comment:
- [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean codebase 😁 😉